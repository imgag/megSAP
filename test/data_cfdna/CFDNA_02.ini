name_short = "cfdna"
name_manufacturer = "cfdna"
target_file = "roi_CFDNA_02.bed"
adapter1_p5 = "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA"
adapter2_p7 = "AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT"
shotgun = 1
umi_type = "IDT-xGen-Prism"
type = "cfDNA"
build = "GRCh38"
