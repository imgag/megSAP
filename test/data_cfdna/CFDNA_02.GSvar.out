##ANALYSISTYPE=SOMATIC_SINGLESAMPLE
##PIPELINE=megSAP 0.2-592-ga3994fe
##CREATION_DATE=2021-06-04
##FILTER=Clustered_Variant=Clustered variants
##FILTER=Error=Alternative counts inside the expected error rate distribution
##FILTER=FDR=no description available
##FILTER=Fisher_Strand=Strand bias based on fisher test
##FILTER=LC_Downstream=Low complexity region (5bps) downstream. ≥ 80% of bases show the same nucleotide  or tandem of ≥ 3 equal nucleotides in a row
##FILTER=LC_Upstream=Low complexity region (5bps) upstream. ≥ 80% of bases show the same nucleotide or tandem of ≥ 3 equal nucleotides in a row
##FILTER=Low_AC=Less than defined minimum of alternative counts
##FILTER=low_conf_region=Variant marked as 'low_conf_region'.
##FILTER=Low_COV=Low coverage
##FILTER=Low_qual_pos=Position enriched with too many low quality bases
##FILTER=off-target=Variant marked as 'off-target'.
##FILTER=PASS=Passed filter
##FILTER=Strand_imbalanced=All alternative reads found in only one strand
##FILTER=Variant_contamination=Reads supporting other alleles outside of the error rate distribution
##SAMPLE=<ID=CFDNA_02,IsTumor=yes,IsFFPE=no>
##DESCRIPTION=filter=Annotations for filtering and ranking variants.
##DESCRIPTION=quality=Quality parameters - variant quality (QUAL), depth (DP), allele frequency (AF), mean mapping quality of alternate allele (MQM), probability of strand bias for alternate bases as phred score (SAP), probability of allele ballance as phred score (ABP)
##DESCRIPTION=gene=Affected gene list (comma-separated).
##DESCRIPTION=variant_type=Variant type.
##DESCRIPTION=coding_and_splicing=Coding and splicing details (Gene, ENST number, type, impact, exon/intron number, HGVS.c, HGVS.p, Pfam domain).
##DESCRIPTION=coding_and_splicing_refseq=Coding and splicing details based on RefSeq (Gene, RefSeq id, type, impact, exon/intron number, HGVS.c, HGVS.p, Pfam domain).
##DESCRIPTION=regulatory=Regulatory consequence details.
##DESCRIPTION=OMIM=OMIM database annotation.
##DESCRIPTION=ClinVar=ClinVar database annotation.
##DESCRIPTION=HGMD=HGMD database annotation.
##DESCRIPTION=RepeatMasker=RepeatMasker annotation.
##DESCRIPTION=dbSNP=Identifier in dbSNP database.
##DESCRIPTION=1000g=Allele frequency in 1000 genomes project.
##DESCRIPTION=gnomAD=Allele frequency in gnomAD project.
##DESCRIPTION=gnomAD_hom_hemi=Homoyzgous counts and hemizygous counts of gnomAD project (genome data).
##DESCRIPTION=gnomAD_sub=Sub-population allele frequenciens (AFR,AMR,EAS,NFE,SAS) in gnomAD project.
##DESCRIPTION=phyloP=phyloP (100way vertebrate) annotation. Deleterious threshold > 1.6.
##DESCRIPTION=Sift=Sift effect prediction and score for each transcript: D=damaging, T=tolerated.
##DESCRIPTION=PolyPhen=PolyPhen (humVar) effect prediction and score for each transcript: D=probably damaging, P=possibly damaging, B=benign.
##DESCRIPTION=fathmm-MKL=fathmm-MKL score (for coding/non-coding regions). Deleterious threshold > 0.5.
##DESCRIPTION=CADD=CADD pathogenicity prediction scores (scaled phred-like). Deleterious threshold > 10-20.
##DESCRIPTION=REVEL=REVEL pathogenicity prediction score. Deleterious threshold > 0.5.
##DESCRIPTION=MaxEntScan=MaxEntScan splicing prediction (reference bases score/alternate bases score).
##DESCRIPTION=dbscSNV=dbscSNV splicing prediction (ADA/RF score).
##DESCRIPTION=COSMIC=COSMIC somatic variant database anntotation.
##DESCRIPTION=MMSplice_DeltaLogitPSI=MMsplice delta Logit PSI score: variant's effect on the exon inclusion - positive score shows higher exon inclusion, negative higher exclusion rate. A score greater than 2 or less than -2 can be considered strong.
##DESCRIPTION=MMSplice_pathogenicity=MMsplice pathogenicity score: probability of pathogenic effect of the variant on splicing
##DESCRIPTION=SpliceAI=SpliceAI prediction of splice-site variations. Probability of the variant being splice-altering (range from 0-1). The score is the maximum value of acceptor/donor gain/loss of all effected genes.
##DESCRIPTION=NGSD_som_c=Somatic variant count in the NGSD.
##DESCRIPTION=NGSD_som_p=Project names of project containing this somatic variant in the NGSD.
##DESCRIPTION=NGSD_som_vicc_interpretation=Somatic variant interpretation according VICC standard in the NGSD.
##DESCRIPTION=NGSD_som_vicc_comment=Somatic VICC interpretation comment in the NGSD.
##DESCRIPTION=NGSD_hom=Homozygous variant count in NGSD.
##DESCRIPTION=NGSD_het=Heterozygous variant count in NGSD.
##DESCRIPTION=NGSD_group=Homozygous / heterozygous variant count in NGSD with the same disease group.
##DESCRIPTION=classification=Classification from the NGSD.
##DESCRIPTION=classification_comment=Classification comment from the NGSD.
##DESCRIPTION=validation=Validation information from the NGSD. Validation results of other samples are listed in brackets!
##DESCRIPTION=comment=Variant comments from the NGSD.
##DESCRIPTION=gene_info=Gene information from NGSD (inheritance mode, gnomAD o/e scores).
##DESCRIPTION=CMC_genes=Gene symbol from COSMIC Cancer Mutation Census (CMC).
##DESCRIPTION=CMC_MUTATION_ID=COSV identifier of variant from COSMIC Cancer Mutation Census (CMC).
##DESCRIPTION=CMC_disease=diseases with > 1% of samples mutated where disease = Primary site(tissue) / Primary histology / Sub-histology = Samples mutated / Samples tested = Frequency from COSMIC Cancer Mutation Census (CMC).
##DESCRIPTION=CMC_DNDS_ratio=diseases with significant dn/ds ratio (q-value < 5%) from COSMIC Cancer Mutation Census (CMC).
##DESCRIPTION=CMC_mutation_significance=Significance tier of the variant from COSMIC Cancer Mutation Census (CMC).
##DESCRIPTION=CANCERHOTSPOTS_AA_CHANGE=Amino acid change as in original cancerhotspots.org file
##DESCRIPTION=CANCERHOTSPOTS_TOTAL_MUT=Total mutation count in cancerhotspots.org at certain amino acid position.
##DESCRIPTION=CANCERHOTSPOTS_ALT_COUNT=Count of specific amino acid alteration at same position in cancerhotspots.org.
##FILTER=low_conf_region=Low confidence region for small variant calling based on gnomAD AC0/RF filters and IMGAG trio/twin data.
##DESCRIPTION=tumor_af=Mutant allele frequency in tumor (Sample CFDNA_02).
##DESCRIPTION=tumor_dp=Tumor Depth (Sample CFDNA_02).
##DESCRIPTION=p-value=Uncorrected p-value
##DESCRIPTION=Strand=Alleles in strands: Alternative forward, Alternative reverse, Reference forward, Reference reverse
##DESCRIPTION=Sequence context=Sequence +/- 5bp around the variant.
##DESCRIPTION=Homopolymer=Indicates if variant is in a homopolymer region.
##DESCRIPTION=somatic_classification_comment=Somatic classification comment from the NGSD.
##DESCRIPTION=somatic_classification=Somatic classification from the NGSD.
#chr	start	end	ref	obs	tumor_af	tumor_dp	p-value	Alt counts	Strand	Sequence context	Homopolymer	filter	quality	gene	variant_type	coding_and_splicing	coding_and_splicing_refseq	regulatory	OMIM	ClinVar	HGMD	RepeatMasker	dbSNP	1000g	gnomAD	gnomAD_hom_hemi	gnomAD_sub	phyloP	Sift	PolyPhen	fathmm-MKL	CADD	REVEL	MaxEntScan	dbscSNV	COSMIC	MMSplice_DeltaLogitPSI	MMSplice_pathogenicity	SpliceAI	NGSD_som_c	NGSD_som_p	NGSD_som_vicc_interpretation	NGSD_som_vicc_comment	NGSD_hom	NGSD_het	NGSD_group	classification	classification_comment	somatic_classification	somatic_classification_comment	validation	comment	gene_info	CMC_genes	CMC_MUTATION_ID	CMC_disease	CMC_DNDS_ratio	CMC_mutation_significance	CANCERHOTSPOTS_AA_CHANGE	CANCERHOTSPOTS_TOTAL_MUT	CANCERHOTSPOTS_ALT_COUNT
chr1	115256530	115256530	G	T	0.0104	4238	0.00019	44	28-16-3034-1160	TTCTTGTCCAG	false		QUAL=36	NRAS	missense	NRAS:ENST00000369535.4:missense_variant:MODERATE:exon3/7:c.181C>A:p.Gln61Lys:PF00071 [Ras family]	NRAS:NM_002524.5:missense_variant:MODERATE:exon3/7:c.181C>A:p.Gln61Lys:		164790 [GENE=NRAS PHENOS=Epidermal nevus,somatic,162900/Melanocytic nevus syndrome,congenital,somatic,137550/Schimmelpenning-Feuerstein-Mims syndrome,somatic mosaic,163200/Colorectal cancer,somatic,114500/RAS-associated autoimmune lymphoproliferative syndrome type IV,somatic,614470/Thyroid carcinoma,follicular,somatic,188470/Neurocutaneous melanosis,somatic,249400/Noonan syndrome 6,613224,Autosomal dominant];	73058 [likely pathogenic(replaced by megSAP based on classifications:1x3,1x5) DISEASE=RAS Inhibitor response,Vascular Tumors Including Pyogenic Granuloma,Hepatocellular carcinoma,Acute myeloid leukemia,Melanoma,Adrenocortical carcinoma,Neuroblastoma,Chronic lymphocytic leukemia,Large congenital melanocytic nevus,Transitional cell carcinoma of the bladder,Multiple myeloma,Renal cell carcinoma, papillary, 1,Malignant melanoma of skin,Lung adenocarcinoma,Non-small cell lung cancer,Neoplasm of brain,Nasopharyngeal Neoplasms,Neoplasm of the large intestine,Adenocarcinoma of stomach,Ovarian Serous Cystadenocarcinoma,Neurocutaneous melanocytosis,Glioblastoma,Rasopathy,Malignant neoplasm of body of uterus,not provided];			rs121913254		0.0000		0.0000,0.0000,0.0000,0.0000,0.0000	9.5240	D(0.01)	P(0.71)	0.99,0.99	26.30	0.84			COSV54736310,COSV54743343,COSV54752117	-0.0253	0.2994	0	78	ALM_Schleimhautmelanom,Leuk,Novartis,OvCa,Ringversuch_EMQN,Roecken_CheckpointInhibitor,SomaticAndTreatment,TEST_SomaticAndTreatment,eMed-HCC	ONCOGENIC		0	2	0 / 0		"[import GPD] Christopher Schroeder 2013-09-09 11:18:14%0A-type: lit / general%0A-link: http://jco.ascopubs.org/content/31/26/3173?cmpid=jco_etoc_10September2013%0A-doi: 10.1200/JCO.2013.50.3144%0A-drugs: imatinib%0AAnother Option in Our KIT of Effective Therapies for Advanced Melanoma%0A=> Of 24 evaluable patients, seven achieved a partial response (PR) to therapy, with five patients' responses confirmed on subsequent imaging studies, for an overall confirmed response rate (RR) of 21%. This clinical activity is similar to that observed in two other trials of imatinib in patients with melanoma molecularly selected for the presence of KIT alterations. Our group3 screened 295 patients for KIT mutations or amplification, identified 51 patients with such alterations, treated 25 evaluable patients, and observed a confirmed RR of 16%, with responses durable for longer than 1 year. Guo et al4 screened 501 patients for KIT mutations or amplification, treated 43 patients with imatinib, and observed an RR of 23.3%. The significance of the responses observed on these three trials is most evident when placed in the context of three prior negative phase II trials of imatinib in molecularly unselected melanoma patients initiated before the identification of KIT mutations in this disease, in which only one radiographic response was observed in 62 patients treated across the three trials.5–7%0A%0A=> Although approximately 70% of KIT mutations identified in melanoma affect the juxtamembrane domain (exon 11, most commonly L576P) and the proximal kinase domain (exon 13, most commonly K642E), others have been identified that likely represent passenger mutations of no biologic significance. %0A=> NRAS Q61K mutation plus a previously detected W557G KIT mutation was also identified in a secondarily resistant tumor after a 7-month course of sunitinib, suggesting that NRAS mutations may also be a mechanism of secondary resistance to KIT inhibition.12,14%0A[3] ahwittd1 08.04.2020%0A[5] ahwittd1 08.04.2020%0A[n/a] ahwittd1 08.04.2020"	activating	[activating] ahwittd1 08.04.2020  Hospotmutation.			NRAS (inh=AD oe_syn=1.00 oe_mis=0.53 oe_lof=0.19), CSDE1 (inh=n/a oe_syn=0.96 oe_mis=0.55 oe_lof=0.05)	NRAS	COSV54736310	skin/malignant_melanoma/acral_lentiginous=14/671=2.09%;skin/malignant_melanoma/superficial_spreading=35/476=7.35%;skin/malignant_melanoma/vertical_growth_phase=13/152=8.55%;thyroid/carcinoma/anaplastic_carcinoma=46/1060=4.34%;skin/benign_melanocytic_nevus/NS=11/334=3.29%;large_intestine/adenoma/NS=4/348=1.15%;thyroid/carcinoma/follicular_carcinoma=28/799=3.5%;thyroid/adenoma-nodule-goitre/NS=14/1373=1.02%;skin/malignant_melanoma/spitzoid=5/185=2.7%;skin/malignant_melanoma/nodular=30/375=8%;thyroid/other/neoplasm=11/132=8.33%;skin/benign_melanocytic_nevus/compound=7/117=5.98%;skin/benign_melanocytic_nevus/intradermal=6/87=6.9%;testis/germ_cell_tumour/seminoma=4/307=1.3%;haematopoietic_and_lymphoid_tissue/haematopoietic_neoplasm/acute_myeloid_leukaemia_therapy_related=6/272=2.21%;skin/malignant_melanoma/NS=444/9975=4.45%;haematopoietic_and_lymphoid_tissue/haematopoietic_neoplasm/acute_myeloid_leukaemia=124/9233=1.34%;soft_tissue/rhabdomyosarcoma/embryonal=27/315=8.57%;skin/benign_melanocytic_nevus/congenital=110/299=36.79%;lung/carcinoma/large_cell_carcinoma=4/215=1.86%;haematopoietic_and_lymphoid_tissue/lymphoid_neoplasm/plasma_cell_myeloma=50/1230=4.07%;NS/malignant_melanoma/NS=78/2109=3.7%	melanoma=3.09e-93;thyroid=2.47e-15;colon=2.8e-20	1			
chr1	115256536	115256536	C	T	0.0097	4430	0.00627	43	14-29-3215-1170	TCCAGCTGTAT	false	FDR	QUAL=22	NRAS	missense	NRAS:ENST00000369535.4:missense_variant:MODERATE:exon3/7:c.175G>A:p.Ala59Thr:PF00071 [Ras family]	NRAS:NM_002524.5:missense_variant:MODERATE:exon3/7:c.175G>A:p.Ala59Thr:		164790 [GENE=NRAS PHENOS=Epidermal nevus,somatic,162900/Melanocytic nevus syndrome,congenital,somatic,137550/Schimmelpenning-Feuerstein-Mims syndrome,somatic mosaic,163200/Colorectal cancer,somatic,114500/RAS-associated autoimmune lymphoproliferative syndrome type IV,somatic,614470/Thyroid carcinoma,follicular,somatic,188470/Neurocutaneous melanosis,somatic,249400/Noonan syndrome 6,613224,Autosomal dominant];	40473 [likely pathogenic DISEASE=not provided];			rs730880965					7.5490	T(0.07)	P(0.69)	0.99,0.99	25.00	0.87			COSV54743124,COSV54743482	0.0053	0.3008	0.01	0				0	0	0 / 0							NRAS (inh=AD oe_syn=1.00 oe_mis=0.53 oe_lof=0.19), CSDE1 (inh=n/a oe_syn=0.96 oe_mis=0.55 oe_lof=0.05)	NRAS	COSV54743124			Other			
